confgenerator/confgenerator.go

```diff
@@ -373,7 +373,7 @@ func (l *Logging) generateFluentbitComponents(userAgent string, hostInfo *host.I
 			out = append(out, s.components...)
 		}
 		if len(tags) > 0 {
-			out = append(out, stackdriverOutputComponent(strings.Join(tags, "|"), userAgent))
+			out = append(out, stackdriverOutputComponent(strings.Join(tags, "|"), userAgent, "20MB"))
```

Comment:
Formatting-wise, I think this should be `20M` according to the docs.

But that aside, if I understand this correctly, a limit of 20 MB means that if a customer loses API connectivity for even one second when logging at a rate of 20 MB/s, they'd start to drop logs? Why are we choosing such a low limit?

Reply:
The docs goes a bit into more details, but this number is number of chunks, not effectively MB. Which is weird because chunks can have different sizes... but that's how fluent-bit implemented it.

Reply:
The code seems to indicate that it's bytes, not chunks: https://github.com/fluent/fluent-bit/blob/caf63dd59dd3eecb66d96b6f969cb30143ad447e/include/fluent-bit/flb_output.h#L370, https://github.com/fluent/fluent-bit/blob/caf63dd59dd3eecb66d96b6f969cb30143ad447e/tests/internal/input_chunk.c#L230